etc/servoTestFrameRate_video.py

```diff
@@ -0,0 +1,124 @@
+import sys
+import os
+import importlib.util
+
+module_path = os.path.join(os.path.dirname(__file__), 'aw', 'commonFunction.py')
+spec = importlib.util.spec_from_file_location("aw", module_path)
+commonFunction = importlib.util.module_from_spec(spec)
+sys.modules["aw"] = commonFunction
+spec.loader.exec_module(commonFunction)
+
+import time
+import subprocess
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.options import ArgOptions
+from selenium.webdriver.common.action_chains import ActionChains
+from selenium.common.exceptions import NoSuchElementException
+
+
+WEBDRIVER_PORT = 7000
+SERVO_URL = f"http://127.0.0.1:{WEBDRIVER_PORT}"
+def setup_hdc_forward():
```

Comment:
It looks like this function is the same in every file - would it make sense to move it to the common module?